gulpfile.js

```diff
@@ -9,13 +9,13 @@ var del = require('del');
 gulp.task('dist', function() {
   gulp.src([ 'src/**/*.coffee' ])
   .pipe(coffee({bare: true}).on('error', gutil.log))
-  .pipe(gulp.dest('./'));
+  .pipe(gulp.dest('./dist'));
 });
 
 gulp.task('coffee', function() {
```

Comment:
Good catch. Yes, let's delete one.
1. Do we want the `bare` option set to `true`? Your [templates](https://github.com/rentpath/generator-rentpath-flight/blob/dev/app/templates/gulpfile.js) have that set to `true`, which compiles the scripts without a top-level function wrapper.
2. Do we want to use the `header` option, which prepends `// Generated by CoffeeScript 1.9.0` to the top of each file. This is useful for development, but not so useful in QA and production. Maybe AG's build process strips out these comments in production. Do you know if it does?